### PR TITLE
#1065 added online check

### LINKED ISSUE #1065: disable invoker tests if we are offline
Without Internet connection our build fails, at the `invoker` step of `eo-maven-plugin`. Let's find a way to disable the entire invoker if we are offline.

### PR DESCRIPTION
**Pull request by mximp**
closes #1065

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256 please merge.

File: eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java [hunk 1]
```diff
@@ -295,6 +297,24 @@ public List<Tojo> select(final Predicate<Tojo> filter) {
         };
     }
 
+    /**
+     * Check the internet connection.
+     *
+     * @return Is the internet connection available
+     */
+    protected static boolean online() {
```
Comment (yegor256) [inline eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java:305]: @mximp we have `WeAreOnline` class already, check it out

Comment (mximp) [reply eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java]: @yegor256 made some refactoring around `WeAreOnline`.

**Comment by mximp** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**